### ISSUE TITLE
InvalidArgumentError (see above for traceback): logits and labels must be same size: logits_size=[9,2] labels_size=[10,2]

### ISSUE THREAD

**Issue by nodegin** [perseus784/BvS#4]
Hi, thanks for this great repo at all. When I start training my model this error will be raised randomly.

`InvalidArgumentError (see above for traceback): logits and labels must be same size: logits_size=[9,2] labels_size=[10,2]`

I have no idea what goes wrong. I am using Python3 with TF 1.2.0

**Comment by perseus784** [comment]
Can you paste the entire error message?. It is required to know which part of the code created the issue. 
As a side note, please check the config file if you have changed any settings.

**Comment by perseus784** [comment]
I assume it is resolved by now and closing the issue
